common/test/db_fixtures/grades.json

```diff
@@ -0,0 +1,11 @@
+[
+    {
+        "pk": 1,
+        "model": "grades.persistentgradesenabledflag",
+        "fields": {
+            "enabled": true,
+            "enabled_for_all_courses": true,
+            "change_date": "2016-09-01"
+        }
+    }
+]
```

Comment:
How do we know that this worked?  Meaning, how do we know that the bokchoy tests are now running with the feature enabled?